js/background.js

```diff
@@ -1,10 +1,24 @@
 
 const TIMER_NAME = "timer";
 
-var runRefresh = function(cfg) {
+function loadTabDetailsAfterLoad(tab, callback) {
+  chrome.tabs.query({windowId: tab.windowId, index: tab.index}, function (tabs) {
```

Comment:
Many people consider usage of arrow functions an improvement as in some JS runtimes, function will cause the actual creations of a Function object (I'm pretty sure though that v8 will optimise this, but...). When writing es6, I tend to use arrows in anonymous places if I don't need this.